tests/lib/managers/files-test.js

```diff
@@ -666,6 +666,57 @@ describe('Files', function() {
 		});
 	});
 
+	describe('uploadFileFromPath()', function() {
+
+		var PARENT_FOLDER_ID = '123',
+			FILENAME = 'folders-test.js',
+			PATH = './folders-test.js';
+
+		var streamFake = {on: sandbox.stub()};
+		it('should call BoxClient.upload() with the correct non-callback params', function() {
```

Comment:
This test is not even testing the method you say it is!  You're calling files.uploadFile() in the test, not files.uploadFileFromPath()